### ISSUE TITLE
Disable or remove "Generate event creator coder table" button

### ISSUE THREAD

**Issue by davidskalinder** [davidskalinder/mpeds-coder#123]
As discussed in #122, this button seems to crash Apache's Python instance when run from any MAI whose `coder_event_creator` table (probably) reaches a certain size, and so amounts to a "temporarily crash server" button.

Both of the live DBs now seem to have reached that point, and so this button only accomplishes two things that I can think of:

1. It allows non-developer users to export the coder-level annotation tables from new or testing MAI deployments
2. It adds the thrill of danger to our mundane lives

So unless we are really desperate for excitement, I don't think there's any reason to leave it in.  Whether to rebuild it properly somewhere else is another question so I'll open another issue for that.

**Comment by davidskalinder** [comment]
@alexhanna, do you have any thoughts about whether it'd be better to simply take the HTML out of the template or to also remove the soon-to-be-unused logic from the JS and controller files?  Here are some ups and downs I can think of about removing AMAP of the underlying logic:

- Pros
  - Removes cruft and confusing dead-end code
  - Can get it back from the git history if we really need it
- Cons
  -  More likely to accidentally break some other thing
  - Easy to re-enable if we want to do that someday
  - Possibly maybe sorta easier to change it into something else that's better

**Comment by davidskalinder** [comment]
> simply take the HTML out of the template

This part is done at 8248e271a1ee4 btw

**Comment by davidskalinder** [comment]
@alexhanna sez cut baby cut.

**Comment by davidskalinder** [comment]
Hey @alexhanna another quick question for you on this: it looks like `generate-coder-table.py` queries the table 1000 rows at a time and writes the output one row at a time, whereas the code in the controller queries the whole table and writes the whole result all at once.

Do we know whether running the chunked approach from the UI button would prevent it from crashing?

If so, then it seems like I should only disable the button HTML and get rid of the bad controller code, leaving the JS as it is so it'll be easy to revive by just swapping the chunked function into the controller?

**Comment by davidskalinder** [comment]
> Do we know whether running the chunked approach from the UI button would prevent it from crashing?

We almost certainly do now: https://github.com/davidskalinder/mpeds-coder/issues/122#issuecomment-846282279

> If so, then it seems like I should only disable the button HTML and get rid of the bad controller code, leaving the JS as it is so it'll be easy to revive by just swapping the chunked function into the controller?

If that.  It might turn out to be easy enough to get the chunked export to work that we can just leave the button there.  This should depend on #122 and comment out the button if #122 gets paused or leave it in and forget about this issue if #122 results in a working export.
